### DISCUSSION TITLE
opencv-python-inference-engine

### DISCUSSION THREAD

**Discussion by trailblazn** [albumentations-team/albumentations#1168] [Q&A] [answered]
Would it be acceptable to add [opencv-python-inference-engine](https://github.com/banderlog/opencv-python-inference-engine) as an acceptable requirement choice in the setup.py file?

Apparently, our data guys insist we need this **_albumentations_** package, and I would like it to work with the **_opencv-python-inference-engine_** package.

Thanks!

**Comment by Dipet** [comment, marked as answer]
Hi. This is an unofficial package and as I can see it is not very popular right now. So I don't think adding this requirement is a good idea.
If you have any conflicts, you can install albumentations with the `--no-deps` flag and install other libraries manually.

**Reply by trailblazn** [reply]
Thank you for the response, that makes sense! I did not know about the `--no-deps` option, that could be useful.
